zerologr.go

```diff
@@ -16,8 +16,7 @@
 // Levels in logr correspond to custom debug levels in Zerolog.  Any given level
 // in logr is represents by `zerologLevel = 1 - logrLevel`.
 // For example V(2) is equivalent to Zerolog's TraceLevel, while V(1) is
-// equivalent to Zerolog's DebugLevel.  Zerolog's usual "level" field is
-// disabled globally and replaced with "v", whose value is a number and is only
```

Comment:
The removal of "Zerolog's usual "level" field is disabled globally" implies that it is no longer disabled. But I don't see such a change in this PR. Was that already done earlier, without updating this doc comment?

Reply:
Looks like https://github.com/go-logr/zerologr/pull/7 did that, so this doc update is just a follow-up.